examples/workflow_example_cep.jl

```diff
@@ -1,46 +1,55 @@
 # This file exemplifies the workflow from data input to optimization result generation
-using ClustForOpt
+using CEP
 using Clp
 ## LOAD DATA ##
-state="GER_1" # or "GER_18" or "CA_1" or "TX_1"
+state="GER_18" # or "GER_18" or "CA_1" or "TX_1"
+years=[2015] #2016 works for GER_1 and CA_1, GER_1 can also be used with 2006 to 2016 and, GER_18 is 2015 TX_1 is 2008
 # laod ts-data
-ts_input_data, = load_timeseries_data("CEP", state; K=365, T=24) #CEP
+ts_input_data = load_timeseries_data_provided(state;T=24, years=years) #CEP
```

Comment:
ending _provided to distinguish between data_path and state

Reply:
Never mind, comment from yesterday, I think we fixed this in the meantime.